src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getAllCustomersValidator = () => {
+    return celebrate({
+      [Segments.QUERY]: Joi.object().keys({
+        offset: Joi.string(),
```

Comment:
O campo `offset` poderia ser um número inteiro aqui, e com valor padrão caso o usuário não informasse:

```suggestion
        offset: Joi
            .number()
            .integer()
            .min(0)
            .default(0),
```